charts/aws-efs-csi-driver/values.yaml

```diff
@@ -110,6 +112,7 @@ node:
     #   cpu: 100m
     #   memory: 128Mi
   nodeSelector: {}
+  updateStrategy: {}
```

Comment:
```suggestion
  updateStrategy:
    {}
    # Override default strategy (RollingUpdate) to speed up deployment. 
    # This can be useful if helm timeouts are observed.
    # type: OnDelete
```

Reply:
updated

Reply:
here lgtm